### PR TITLE
Examples for Either.bimap, Either.fold and Either.traverse (#2458)

### PR DESCRIPTION
**Pull request by zbyszkop**
Continuing with PRs for Either - code examples in javadoc for bimap and fold

### CODE REVIEW

File: src/main/java/io/vavr/control/Either.java [hunk 1]
```diff
@@ -193,6 +193,18 @@ private Either() {
     /**
      * Maps either the left or the right side of this disjunction.
      *
+     * <pre>{@code
+     *Either<?, AtomicInteger> success = Either.right(new AtomicInteger(42));
```
Comment (danieldietrich) [inline src/main/java/io/vavr/control/Either.java]: one space missing at the beginning

File: src/main/java/io/vavr/control/Either.java [hunk 2]
```diff
@@ -193,6 +193,18 @@ private Either() {
...
+     *
+     * //prints "Right(42)"
```
Comment (danieldietrich) [inline src/main/java/io/vavr/control/Either.java]: Please add one space after `//` as we do it across the whole code base for reasons of consistency and readability.

Comment (zbyszkop) [reply src/main/java/io/vavr/control/Either.java]: Fixed, also in previous change.

File: src/main/java/io/vavr/control/Either.java (hunk 1 above)
Comment (zbyszkop) [reply src/main/java/io/vavr/control/Either.java]: Fixed.

**Pull request merged**